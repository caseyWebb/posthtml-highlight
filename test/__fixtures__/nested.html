<pre><code>
  const foo = 'foo'
  console.log(foo)
  <small>
    const bar = 'bar'
    console.log(bar)
  </small>
</code></pre>
