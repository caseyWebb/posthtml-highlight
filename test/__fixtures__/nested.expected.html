<pre><code class="hljs">
  <span class="hljs-keyword">const</span> foo = <span class="hljs-string">'foo'</span>
  console.<span class="hljs-built_in">log</span>(foo)
  <small class="hljs">
    <span class="hljs-keyword">const</span> bar = <span class="hljs-string">'bar'</span>
    console.<span class="hljs-built_in">log</span>(bar)
  </small>
</code></pre>
